osu.Game/Rulesets/Judgements/JudgementResult.cs

```diff
@@ -30,6 +30,8 @@ public class JudgementResult
         [NotNull]
         public readonly Judgement Judgement;
 
+        public HitDetail Detail => TimeOffset < 0 ? HitDetail.Early : HitDetail.Late;
```

Comment:
What happens when hitting at perfect timing `TimeOffset = 0`?

Reply:
Just realized that no detail text appears on `Great` judgement, although I advocate for adding a `Perfect` detail rather than having a false value.

Reply:
It's hidden when `TimeOffset` is within the window of a Perfect or Great judgement, because the judgement text already gives the information that we hit perfectly.

Reply:
Alright, actually done that at first but could not find a use-case where it would be used. If you think it's more fitting to have a third member in the Enum I'll be adding one